### DISCUSSION TITLE
Wydłużenie czasu działania metod WzA parcel-management

### DISCUSSION THREAD

**Discussion by q3ok** [allegro/allegro-api#9220] [Suggestions - resolved]
W nawiązaniu do
https://github.com/allegro/allegro-api/issues/9214
https://github.com/allegro/allegro-api/discussions/9218
https://github.com/allegro/allegro-api/discussions/9217

Uprzejmie proszę o przedłużenie czasu działania metod WzA parcel-management.

**Comment by MartaNowaczyk** [comment]
Przedłużono czas działania do 31.01.2025.
